core/src/main/scripts/importer/validateData.py

```diff
@@ -1375,6 +1390,36 @@ def checkEndPosition(self, value):
         # if no reasons to return with a message were found, return valid
         return True
 
+    def checkDriverPassenger(self, value):
+        """Validate the values in the binary column."""
+        if value not in ('PUTATIVE_PASSENGER', 'PUTATIVE_DRIVER'):
```

Comment:
The MAF format usually has their values in title case, such as Missense_Mutation and Unknown. For consistency, we could have the examples also in this format, such as Putative_Passenger and Unknown. But case sensitivity is kind of annoying, so it would also be nice if the all values are validated and imported case insensitive, but saved in a fixed format.